### DISCUSSION TITLE
path integral molecular dynamics using i-PI

### DISCUSSION THREAD

**Discussion by lnnbig** [deepmodeling/deepmd-kit#1712] [Q&A]
Can we train an NNP using a training set generated from classical (DFT) MD, then use this NNP for path integral MD using lammps+i-PI?
Does the training set has to be from path integral rather than classical MD?

**Comment by njzjz** [comment]
> Can we train an NNP using a training set generated from classical (DFT) MD, then use this NNP for path integral MD

Yes, I have a paper about it: https://doi.org/10.1021/acs.jctc.2c00151
